### ISSUE TITLE
Improper use of array_keys()

### ISSUE THREAD

**Issue by curlyjason** [OrigamiStructures/StudioManagerScratch#155]
On line 69 of AdminstratorController:

`			$editionIDs = array_keys($editions
						->find()
						->specifyFilter('artwork_id', $artworkID)
						->load());
`

The load() function returns an object. the array_keys() function requires an array.

**Comment by dreamingmind** [comment]
Any idea when this gets run? I don’t see a usage for the method.

Both my memory of load and the hinting on this code point to it returning an array as required. 

But if you remember how you came across the error I’ll check it out.

**Comment by dreamingmind** [comment]
I found when this runs (submit the form on administrator/index) and can get an error on the ‘Leonardo’ choice. But my error is:

Call to a member function IDs() on array

So, I’m seeing the same problem as you are. 

It’s possible you got to the method from a different call point or that we have some different date and so we hit different edge cases.

At any rate, I tracked down the cause and solution of the error I found.

Also, I have a clue about the purpose of this action and it is both important and live.

I was exploring a fundamental change to the Format handling in Artworks (meant to reduce the user's repetitive data entry tasks).

This was/is a dangerous change and it appears this Administrator controller was a data validation routine.

D
